tests/roots/test-resolve-typing-guard/demo_typing_guard_dummy.py

```diff
@@ -0,0 +1,5 @@
+from viktor import AI  # 'viktor' part of autodoc_mock_imports
```

Comment:
What is autodoc_mock_imports?

Reply:
Ping on this :blush:

Reply:
External dependencies can be mocked when specified in the conf.py: https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_mock_imports